src/components/SButton.vue

```diff
@@ -0,0 +1,15 @@
+<script>
+export default {
+  name: 'SButton'
+}
+</script>
+
+<template>
+    <button>
+        <slot name="value"/>
+    </button>
+</template>
```

Comment:
Make sure your indents are set properly. It looks like your script is indented by 2 and your html is indented by 4. It should be consistent in all areas, including CSS.